app/src/main/res/values/strings.xml

```diff
@@ -508,7 +508,7 @@
     <string name="timebase">TimeBase</string>
     <string name="timebase_trigger_value">875.00 μs</string>
     <string name="text_view">TextView</string>
-    <string name="pslab_device_not_found">PSLAB Device not found</string>
+    <string name="pslab_device_not_found">PS Lab Device not found</string>
     <string name="select_mic">[SELECT MIC]</string>
```

Comment:
I think it should be PSLab. It is the correct name. Check [here](https://pslab.fossasia.org/)